src/commands/Fun/wakanda.ts

```diff
@@ -22,7 +23,7 @@ export default class extends SkyraCommand {
 		const userAvatar = await fetchAvatar(user);
 		const image = this.generateImage(userAvatar);
 
-		return message.channel.sendFile(image.toBuffer(), "we-don't-do-that-here.png");
+		return message.channel.sendFile(image.toBuffer(), 'we-do-not-do-that-here.png');
```

Comment:
Not sure why you removed the apostrophe here? The original quote uses don't, and so should the file name.

Reply:
It's a special character, which can lead to issues in some OSes.

Reply:
Also renamed the source png now